ksqldb-parser/src/main/java/io/confluent/ksql/util/ParserKeywordValidatorUtil.java

```diff
@@ -26,9 +26,10 @@
 /**
```

Comment:
Could you move the comment to the method used for checking the function name and change the class javadoc to be that the class provides methods for checking whether keywords are java or ksql reserved?

Reply:
done